packages/kununu-footer/index.jsx

```diff
@@ -61,10 +67,10 @@ export default class Footer extends Component { // eslint-disable-line
       <footer
         role="banner"
         id="footer"
-        className={`navbar-default ${styles.footer}`}
+        className={`navbar-default ${styles.footer} ${simpleMobile ? styles.simpleMobile : ''}`}
```

Comment:
please use the classnames package here

Reply:
Since we are not yet using the classNames package in client-gear I am not sure I want to introduce it with such small ternary. I think we should bring it in only once we really need it.. if for instance we have complex classes being added.